chsdi/views/loader.py

```diff
@@ -50,7 +57,7 @@ def get_resource_url(filename, extension, mode_str=''):
             'ga_js': ga_js,
             'epsg_21781_js': epsg_21781_js,
             'epsg_2056_js': epsg_2056_js,
-            'api_url': request.path_url.replace('/loader.js', ''),
+            'api_url': api_url.replace('/loader.js', ''),
```

Comment:
Dito, not sure about this adaption here

Reply:
This is not good, the API url refer to the mf-chsdi3 API url not to the bucket URL. But the original code has also an issue, it doesn't take into account the Forwarded Proto header ! So it will come with http instead of https for the official api call (not hproxy).

Reply:
But then I could simply adapt line 50 as follows:
`api_url = host` (or `host_url`), as defined in the if.. elif.. above
Then it should use the correct proto and only the `host_url`

Reply:
But as I understand it, currently the `'api_url': request.path_url.replace('/loader.js', '')` also is not only restricted to the API url, but has the full request path in it